util/events/libuv/Process.c

```diff
@@ -18,6 +18,11 @@
 #include "util/Bits.h"
 #include "util/Identity.h"
 
+void Process_OnExitCallbackWrapper(long pid, int exitCode) {
+    long long pidLongLong = (long long) pid;
```

Comment:
Should be casted to an `int64_t` because that is not guaranteed to be a `long long`. On 64 bit machines it is usually a `long` and I believe on some machines it is possible that `long long` is `uint128_t`.